server/src/internalClusterTest/java/org/elasticsearch/search/profile/dfs/DfsProfilerIT.java

```diff
@@ -76,7 +75,6 @@ public void testProfileDfs() throws Exception {
             if (randomBoolean()) {
                 knnSearchBuilder.addFilterQuery(q);
             }
-            logger.info("Query: {}", q);
```

Comment:
would there be a way to add a more specific test that reproduces this concurrency issue? Perhaps a unit test?

Reply:
I mean, sure, there are always ways. I don't see the benefit if this test reliably fails. 

Do you have an idea of an existing test suite or class where this would belong?

Reply:
I am not entirely sure about the interaction between the profiler and the search, but possibly it would fit into ContextIndexSearcherTests, or a new test suite that verifies the profiler behaviour. I would prefer that rather than relying on an IT test to randomly fail, for a known issue.